build_scripts/windows/promptflow.wixproj

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Project -->
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProductVersion>3.10</ProductVersion>
+    <ProjectGuid>04ff6707-750d-4474-89b3-7922c84721be</ProjectGuid>
+    <SchemaVersion>2.0</SchemaVersion>
+    <OutputName>Promptflow</OutputName>
```

Comment:
Let's use all lower case for file/directory names: promptflow

Reply:
done